sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/DeleteRepositoryResult.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.containers.containerregistry.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+/** Deleted repository. */
+@Fluent
+public final class DeleteRepositoryResult {
+    /*
+     * SHA of the deleted image
+     */
+    @JsonProperty(value = "manifestsDeleted")
+    private List<String> deletedRegistryArtifactDigests;
+
+    /*
+     * Tag of the deleted image
+     */
+    @JsonProperty(value = "tagsDeleted")
+    private List<String> deletedTags;
```

Comment:
Service question: Should this be made into a type as it appears to be associated lists. Possibly `DeletedRepositoryMetadata` or something like that

Reply:
Good point. I will look through the usage patterns.

---
In reply to: [602633214](https://github.com/Azure/azure-sdk-for-java/pull/20139#discussion_r602633214) [](ancestors = 602633214)